controllers/flinkcluster_converter.go

```diff
@@ -330,43 +312,19 @@ func getDesiredTaskManagerDeployment(
 	var rpcPort = corev1.ContainerPort{Name: "rpc", ContainerPort: *taskManagerSpec.Ports.RPC}
 	var queryPort = corev1.ContainerPort{Name: "query", ContainerPort: *taskManagerSpec.Ports.Query}
 	var taskManagerDeploymentName = getTaskManagerDeploymentName(clusterName)
-	var jobManagerDeploymentName = getJobManagerDeploymentName(clusterName)
 	var labels = map[string]string{
 		"cluster":   clusterName,
 		"app":       "flink",
 		"component": "taskmanager",
 	}
-	var envVars = []corev1.EnvVar{
-		{
-			Name:  "JOB_MANAGER_RPC_ADDRESS",
-			Value: jobManagerDeploymentName,
-		},
-		{
-			Name: "TASK_MANAGER_CPU_LIMIT",
-			ValueFrom: &corev1.EnvVarSource{
-				ResourceFieldRef: &corev1.ResourceFieldSelector{
-					ContainerName: "taskmanager",
-					Resource:      "limits.cpu",
-					Divisor:       resource.MustParse("1m"),
-				},
-			},
-		},
-		{
-			Name: "TASK_MANAGER_MEMORY_LIMIT",
```

Comment:
Previously, I was using downward API to get CPU and memory limit, then in my custom image, I used the env variables to update the heapSize and numOfSlots in flink config. I plan to make a upstream PR, to allow passing the values to the official image. So, here can we keep the TASK_MANAGER_CPU_LIMIT and TASK_MANAGER_MEMORY_LIMIT?

Reply:
Yes, I will preserve them.